### PR TITLE
Update developers-clear-containers-install.md

### PR DESCRIPTION
**Pull request by mcastelino**
Add kernel debugging documentation.

### CODE REVIEW

File: docs/developers-clear-containers-install.md [hunk 1]
```diff
@@ -168,3 +168,5 @@ $ make help
 
   * [General Debugging](../README.md#debugging)
   * [Debugging the agent inside the hypervisor](debug-agent.md)
+  * [Debugging the kernel inside the hypervisor](https://github.com/mcastelino/runtime/blob/topic/docs/docs/debug-kernel.md)
```
Comment (gorozco1) [inline docs/developers-clear-containers-install.md]: Could you update the link to be the one in `clearcontainers` repo not your private repo @mcastelino

Comment (gorozco1) [reply docs/developers-clear-containers-install.md]: Oh I just recall that this is udpated automatically with the merge..

**Comment by gorozco1** [comment]
lgtm

File: docs/developers-clear-containers-install.md (hunk 1 above)
Comment (jodh-intel) [reply docs/developers-clear-containers-install.md]: @gorozco1 - I think your original comment is correct - the doc already exists in the runtime repo so the doc needs to reference:

```
https://github.com/clearcontainers/runtime/blob/master/docs/debug-kernel.md
```

**Comment by gorozco1** [comment]
any update on this?

File: docs/developers-clear-containers-install.md (hunk 1 above)
Comment (iphutch) [reply docs/developers-clear-containers-install.md]: Once the link is updated to a clearcontainers repo I am happy to approve.

**Comment by mcastelino** [comment]
@iphutch @gorozco1 fixed the link

**Comment by gorozco1** [comment]
👍

**Comment by jodh-intel** [comment]
Doc-only change so merging...

**Pull request merged**